whois-api/src/test/java/net/ripe/db/whois/api/httpserver/RemoteAddrCustomizerNonTrustedTestIntegration.java

```diff
@@ -0,0 +1,38 @@
+package net.ripe.db.whois.api.httpserver;
+
+import com.google.common.net.HttpHeaders;
+import net.ripe.db.whois.api.AbstractIntegrationTest;
+import net.ripe.db.whois.api.RestTest;
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsString;
+
+@Tag("IntegrationTest")
+public class RemoteAddrCustomizerNonTrustedTestIntegration extends AbstractIntegrationTest {
```

Comment:
shall we merge this test case with "RemoteAddressCustomizerTestIntegration"?
It seems that in this test case we only can test one thing.
"the remote is not trusted then take the X_FORWARDED_FOR"
This make me think that maybe this should go in a more generic test class.

Btw we are going to remove X_FORWARDED_FOR, so do we need a test case for it? If we enable client auth X_FORWARDED_FOR will not longer be working...

Reply:
We cannot  merge this test case with "RemoteAddressCustomizerTestIntegration" as trusted ipRanges are set during spring initialization through system.setProperty hence we need a separate test class to test this scenario